webpack.config.js

```diff
@@ -2,14 +2,19 @@ const path = require('path');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 
-const genConfig = (filename, { inlineCss } = {}) => ({
+const genConfig = (outputFilename, { inlineCss, inputFilename = 'index.js' } = {}) => ({
   output: {
     path: path.resolve(__dirname, 'dist'),
-    filename,
+    filename: outputFilename,
     library: 'superheroUtils',
     libraryTarget: 'umd'
   },
+  entry: `./src/${inputFilename}`,
   target: 'web',
+  externals: {
+    'prop-types': 'prop-types',
+    'react': 'react'
```

Comment:
Actually, this is not working if imported using the `script` tag because React exposes itself there as `React`, it is fixed in #38